### ISSUE TITLE
Update Readme.md

### ISSUE THREAD

**Issue by chamindra** [chamindra/innersource-license-generator#14]
Update the description and purpose and references 

Provide clear disclaimers on as-is clauses and warrenty

Provide risk advisory

**Comment by spier** [comment]
How about adding a link to the InnerSource Commons Summit 2023 presentation to the description?
This could provided further context about goals/purpose etc.

**Comment by chamindra** [comment]
Good idea. I will also wait for the video to be published and then we can include it.
